workspaces/config/lib/definitions/definitions.js

```diff
@@ -2012,11 +2012,14 @@ const definitions = {
       If you ask npm to install a package and don't tell it a specific version,
       then it will install the specified tag.
 
-      Also the tag that is added to the package@version specified by the \`npm
-      tag\` command, if no explicit tag is given.
+      Also the tag that is added to the package@version specified by the 
+      \`npm dist-tag add\` command, if no explicit tag is given.
```

Comment:
The grammar of this sentence was not and still is not correct. 

"The tag is also added to ... "

Reply:
I read it several times and thought something was wrong but wasn't 100% sure as not a native English speaker.

`The tag is also added` for me sounds like same tag is added in two places, whilst they're describing two different scenarios (what happens for the config / CLI arg in `npm i` vs `npm dist-tag add`.

Updated it to be:

> It is the tag added to the package@version specified in the `npm dist-tag add` command, if no explicit tag is given.

Changed also the `by` -> `in` as the users specifies the `package@version` to the command (it's not the command the one specifying that string)

Though suggestions very welcome ofc